builtin/logical/totp/backend.go

```diff
@@ -25,11 +27,15 @@ func Backend(conf *logical.BackendConfig) *backend {
 		Secrets: []*framework.Secret{},
 	}
 
+	b.usedCodes = cache.New(0, 1*time.Second)
```

Comment:
I also wonder if we can afford to wait a little longer to look for expired items instead of doing it every second.